Jenkinsfile

```diff
@@ -1,5 +1,4 @@
 buildPlugin(useContainerAgent: true, configurations: [
-  [ platform: 'linux', jdk: 11 ],
```

Comment:
if removing 11 does it mean you want the plugin to be only >= 17?

Reply:
Solely changed the JDK used to built the plugin, it'll work with 11 still

Reply:
sure, but this will not be tested anymore. I would prefer to keep here the minimum Java version still supported by the plugin to be sure it's building fine.

Reply:
Java 11 for this plugin is tested regularly by the plugin bill of materials. I think that we can rely on the plugin BOM to assure that Java 11 is still running the tests of this plugin as expected.